client/src/views/Comparison/Comparison.vue

```diff
@@ -1,179 +1,212 @@
 <template>
   <div class="view-container">
-    <div class="search-row">
+    <header>
```

Comment:
I was thinking about it. Maybe we don't really need this one anymore...The Model view is still important because we do there a lot of exploration. I'd say let's keep this as it is for now and we can revisit this later.